Jenkinsfiles/maint-mode.yml

```diff
@@ -0,0 +1,3 @@
+pipeline:
```

Comment:
Hi @safwanrahman! This file is required so that when we force push this branch to the ["maint-mode" branch](https://github.com/mozilla/kuma/tree/maint-mode), we get an automatic deployment to AWS (specifically to https://mdn-demo-maint-mode.virginia.moz.works). We couldn't use this branch name because the resulting name ("mdn-demo-maintenance-mode-1313237" violated a length limit enforced by one of our deployment tools, I think it was Deis?).